### PR TITLE
Add changes to enable debug for ios QA

### PR DESCRIPTION
**Pull request by TheNoodleMoose**
These changes enable debugging for ios to help Clark get through the QA process quicker. @levieggertcru I'd appreciate it if you could take a look just to make sure I did this right 🙂

### CODE REVIEW

File: ios/MissionHub/AppDelegate.m [hunk 1]
```diff
@@ -71,7 +75,8 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   #if FB_SONARKIT_ENABLED
     InitializeFlipper(application);
   #endif
-
+  GAI *gai = [GAI sharedInstance];
+  gai.dispatchInterval = 1;
```
Comment (TheNoodleMoose) [inline ios/MissionHub/AppDelegate.m:79]: @levieggertcru Not sure if I added this in the right spot or not.

Comment (levieggertcru) [reply ios/MissionHub/AppDelegate.m:79]: Hey Christian that should be fine to set it there when application has finished launching.

Comment (TheNoodleMoose) [reply ios/MissionHub/AppDelegate.m:79]: Great, thanks!

File: ios/MissionHub-Bridging-Header.h [hunk 2]
```diff
@@ -2,3 +2,10 @@
 //  Use this file to import your target's public headers that you would like to expose to Swift.
 //
 
+// The Google/Analytics pod has been deprecated. Since moving to the GoogleAnalytics pod, these need to be manually added to the bridging header.
+#import <GoogleAnalytics/GAI.h>
+#import <GoogleAnalytics/GAIFields.h>
+#import <GoogleAnalytics/GAILogger.h>
+#import <GoogleAnalytics/GAIDictionaryBuilder.h>
+#import <GoogleAnalytics/GAITrackedViewController.h>
+#import <GoogleAnalytics/GAITracker.h>
```
Comment (OzzieOrca) [inline ios/MissionHub-Bridging-Header.h:11]: So these are needed? Not sure I understand the comment. The Pod is deprecated but we recently moved to use it?

File: ios/MissionHub.xcodeproj/xcshareddata/xcschemes/MissionHub.xcscheme [hunk 3]
```diff
@@ -60,6 +60,16 @@
...
       </BuildableProductRunnable>
+      <CommandLineArguments>
+         <CommandLineArgument
+            argument = "-FIRDebugEnabled"
+            isEnabled = "YES">
+         </CommandLineArgument>
+         <CommandLineArgument
+            argument = "-FIRDebugDisabled"
+            isEnabled = "NO">
+         </CommandLineArgument>
```
Comment (OzzieOrca) [inline ios/MissionHub.xcodeproj/xcshareddata/xcschemes/MissionHub.xcscheme:71]: This seems duplicate but maybe that's how it works.

**Review by OzzieOrca — APPROVED** [review]

File: ios/MissionHub-Bridging-Header.h (hunk 2 above)
Comment (TheNoodleMoose) [reply ios/MissionHub-Bridging-Header.h:11]: @OzzieOrca Yeah I'm not too sure. I'm basing these changes on how it was done for GodTools on this pr https://github.com/CruGlobal/godtools-swift/pull/619

File: ios/MissionHub.xcodeproj/xcshareddata/xcschemes/MissionHub.xcscheme (hunk 3 above)
Comment (TheNoodleMoose) [reply ios/MissionHub.xcodeproj/xcshareddata/xcschemes/MissionHub.xcscheme:71]: Yeah not sure either on this one. @levieggertcru is this needed for a particular reason?

**Comment by TheNoodleMoose** [comment]
@OzzieOrca Also we'll be removing this once Clark is done QAing btw.

**Pull request merged**